index.js

```diff
@@ -260,7 +265,7 @@ function fastifyView (fastify, opts, next) {
   }
 
   function readCallback (that, page, data, localOptions) {
```

Comment:
This whole logic would have to change because we are not using callbacks anymore, we are using async/await.